### ISSUE TITLE
EnergyPlus does not recognize Zone Sizing is Initialized

### ISSUE THREAD

**Issue by Ski90Moo** [openstudiocoalition/OpenStudioApplication#691]
Issue overview
--------------

This may be and EnergyPlus issue...

When the model is run, the following error appears in the eplusout.err file:
   ** Severe  ** ManageSizing: Requested System Sizing but did not request Zone Sizing.
[HEP-24-03 CALCS_002.txt](https://github.com/openstudiocoalition/OpenStudioApplication/files/14598989/HEP-24-03.CALCS_002.txt)

   **   ~~~   ** System Sizing cannot be done without Zone Sizing
   **  Fatal  ** Program terminates for preceding conditions.

## Current Behavior

OpenStudio does request Zone sizing:

OS:SimulationControl,
  {3761ef35-bae8-4829-aabb-a11ecfac8e0e}, !- Handle
  Yes,                                    !- Do Zone Sizing Calculation
  Yes,                                    !- Do System Sizing Calculation
  Yes,                                    !- Do Plant Sizing Calculation
  Yes,                                    !- Run Simulation for Sizing Periods
  Yes,                                    !- Run Simulation for Weather File Run Periods
  ,                                       !- Loads Convergence Tolerance Value {W}
  ,                                       !- Temperature Convergence Tolerance Value {deltaC}
  ,                                       !- Solar Distribution
  ,                                       !- Maximum Number of Warmup Days
  ,                                       !- Minimum Number of Warmup Days
  Yes;                                    !- Do HVAC Sizing Simulation for Sizing Periods

The in.idf file also does Request Zone sizing:

SimulationControl,
  Yes,                                    !- Do Zone Sizing Calculation
  Yes,                                    !- Do System Sizing Calculation
  Yes,                                    !- Do Plant Sizing Calculation
  Yes,                                    !- Run Simulation for Sizing Periods
  Yes,                                    !- Run Simulation for Weather File Run Periods
  Yes,                                    !- Do HVAC Sizing Simulation for Sizing Periods
  ;                                       !- Maximum Number of HVAC Sizing Simulation Passes

## Steps to Reproduce
OSM file is attached.  Click run.

1.
2.
3.
4.

### Environment

Some additional details about your environment for this issue (if relevant):
 - Platform (Operating system, version):
![image](https://github.com/openstudiocoalition/OpenStudioApplication/assets/69771412/d21539de-914f-48bc-84e8-539c9e4f0d8c)

 - Version of OpenStudioApplication (if using an intermediate build, include SHA):
![image](https://github.com/openstudiocoalition/OpenStudioApplication/assets/69771412/69a63c6d-0e33-4d17-a184-ddeb1687f1f9)

**Comment by macumber** [comment]
I looked at your model @Ski90Moo, I think your current problem is that you need to add diffusers to your zones in the air loop tab

![image](https://github.com/openstudiocoalition/OpenStudioApplication/assets/1327850/2d4f7e52-7a3e-49a5-9449-b7815427a69e)

After I added those to your model I got past the first issue and on to some issues with your geometry.  Hope that helps

**Comment by Ski90Moo** [comment]
Thank you @macumber.  I have encountered this issue before, but I think I confused the diagnosis when I posted this issue.  It appears that the default when creating a new OS model is that Do Zone Sizing Calculation and Do System Sizing Calculation is toggled off.  Why is that?  Without input from the users, I think by default these should be toggled on.  For example, when I import spaces, create thermal zones and thermostats, and add an autosized Packaged Rooftop Air Conditioner and assign zones to the diffusers, these two remain toggled off.  It then gives this error and the user has to diagnose the problem.
